app/HutchManager/Config/RegistrationOptions.cs

```diff
@@ -0,0 +1,8 @@
+namespace HutchManager.Config;
+
+public class RegistrationOptions
+{
+  public const string UserAccounts = "UserAccounts";
```

Comment:
don't think we need this constant?

This class should just represent the values we care about from the related config section, nothing else
```suggestion
```